DigitalOffice/Services/TimeManagementService/Controllers/TimeManagementController.cs

```diff
@@ -1,11 +1,21 @@
 ﻿using Microsoft.AspNetCore.Mvc;
+using System;
+using LT.DigitalOffice.TimeManagementService.Commands.Interfaces;
+using LT.DigitalOffice.TimeManagementService.Models;
 
 namespace LT.DigitalOffice.TimeManagementService.Controllers
 {
-    [Route("[controller]")]
+    [Route("api/[controller]")]
     [ApiController]
     public class TimeManagementController : ControllerBase
     {
-        //TODD Add endpoints.
+        [Route("AddWorkTime")]
+        [HttpPost]
+        public Guid AddWorkTime(
```

Comment:
It's not a suggestion nor a comment, just a thought:
We can also write [HttpPost("AddWorkTime")], that will reduce the code for 1 string.
Spartak approved, so feel free to think about it.

Reply:
Fixed.